app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php

```diff
@@ -85,10 +96,11 @@ public function __construct(
      * Validates that the email address isn't being used by a different account.
      *
      * @param string $email
-     * @throws LocalizedException
+     *
      * @return void
+     * @throws LocalizedException
      */
-    protected function validateEmailAvailable($email)
+    protected function validateEmailAvailable(string $email)
```

Comment:
Please revert it, since it's BIC
```suggestion
    protected function validateEmailAvailable($email)
```